docs/technical_specifications/ONBOARDING.md

```diff
@@ -35,19 +37,49 @@ behalf of that org for the registration of an entity, if not already present wit
             - check of the presence of kid claim
             - check that the certificate is not expired
             - check that the kid is unique and of the lenght of a minimum JWK thumbprint length
-    5. at least one administrative email contact (String)
+    5. TODO: at least one administrative email contact (String)
```

Comment:
Please add it in the registration form and create a Descendat Contact with the admin action